### PR TITLE
Don't import System when it's already imported by global using

### LINKED ISSUE #5652: CA1845: Adds unnecessary using statement when implicit usings exist
**Version Used**: 
Microsoft Visual Studio Enterprise 2022 Int Preview (64-bit)
Version 17.0.0 Preview 5.0 [31808.319.d17.0]

**Steps to Reproduce**:

1. Create a new console app in .NET6 - default implicit usings
2. Add some code that triggers CA1845
3. Executing the code fix

**Expected Behavior**:
`using System;` is not added

**Actual Behavior**:
It is added

Not sure if this is expected with the ImplicitUsings feature, but I just always hate when tools add something immediately grayed out.
implicit

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #5652

### CODE REVIEW

File: src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloads.Fixer.cs [hunk 1]
```diff
@@ -30,12 +30,11 @@ namespace Microsoft.NetCore.Analyzers.Runtime
     /// </summary>
     public abstract class PreferStreamAsyncMemoryOverloadsFixer : CodeFixProvider
     {
+        private static readonly SyntaxAnnotation s_asMemorySymbolAnnotation = new("SymbolId", "System.MemoryExtensions");
```
Comment (Youssef1313) [inline src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStreamAsyncMemoryOverloads.Fixer.cs:33]: This is quite an implementation detail on Roslyn. I have an API proposal to expose it in cleaner way. https://github.com/dotnet/roslyn/issues/65041

**Comment by Youssef1313** [comment]
@CyrusNajmabadi Looks like I got a repro here for the line ending issue. https://github.com/dotnet/roslyn/issues/65081

Can you please take a look? Thanks!

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by Youssef1313** [comment]
My bad. This is an extra line, not line ending difference. This is completely unrelated to dotnet/roslyn#65081.

**Comment by Youssef1313** [comment]
and now ubuntu only failure related to end of lines :smile: 
So it is dotnet/roslyn#65081

**Comment by Youssef1313** [comment]
@mavasani Failure here is a Roslyn bug that we have in other codefixes that uses `AddImportsAnnotation`.

Is it okay to disable the test on Linux? Any workaround you can think of?

**Comment by mavasani** [comment]
> Is it okay to disable the test on Linux?

Yes, that sounds reasonable. I believe we have an attribute to execute tests only on Windows platform that is used by some tests. I can't recall if it exists only in Roslyn or also in this repo.

**Comment by Youssef1313** [comment]
> > Is it okay to disable the test on Linux?
> 
> Yes, that sounds reasonable. I believe we have an attribute to execute tests only on Windows platform that is used by some tests. I can't recall if it exists only in Roslyn or also in this repo.

It exists here and removal is tracked by https://github.com/dotnet/roslyn-analyzers/issues/4586.

**Pull request merged**